README.md

```diff
@@ -190,17 +224,15 @@ In general, every development board has the following basic features:
 <!-- CONTRIBUTORS -->
 ## Contributors
 
-- [Omkar Bhilare](https://github.com/ombhilare999): *Designer*
-- [Dhiraj Patil](https://github.com/dhirajp15): *Mentor*
-- [Saurabh Gupta](https://github.com/saurabh1002): *Mentor*
-- [Udit Patadia](https://github.com/udit7395): *Mentor*
-- [Laukik Hase](https://github.com/laukik-hase): *Mentor*
+- [Dhairya Shah](https://github.com/dhairyashah1): *Designer*
+- [Omkar Bhilare](https://github.com/ombhilare999): *Mentor*
+- [Vedant Paranjpe](https://github.com/VedantParanjape): *Mentor*
 
 <!-- ACKNOWLEDGEMENTS AND REFERENCES -->
 ## Acknowledgements and Resources
 -   Thanks to [OSH](https://oshpark.com/) for providing us with free coupons worth $100 for printing the prototype boards.
```

Comment:
Thanks to [PCBPowerMarket](https://www.pcbpower.com/) for sponsoring the first prototypes.

Replace first line with the above